lib/main_containers.pm

```diff
@@ -143,7 +143,7 @@ sub load_host_tests_podman {
     # Buildah is not available in SLE Micro, MicroOS and staging projects
     load_buildah_tests($run_args) unless (is_sle('<15') || is_sle_micro || is_microos || is_leap_micro || is_staging);
     loadtest('containers/skopeo', run_args => $run_args, name => $run_args->{runtime} . "_skopeo") unless (is_sle('<15') || is_sle_micro('<5.5'));
-    loadtest 'containers/podman_quadlet' if is_tumbleweed;
+    loadtest 'containers/podman_quadlet' if (is_tumbleweed || (is_sle && !is_sle("<16")));
```

Comment:
```suggestion
    loadtest 'containers/podman_quadlet' unless (is_sle("<16") || is_sle_micro("<6.0"));
```
I would simplify this check. Please check if this works, but in general we can run this everywhere except on < SLES16 and SLEM < 6.0 (I think)